### PR TITLE
Setup project branch

### PR DESCRIPTION
**Pull request by nicupop729**
### In this feature branch, the following implementations were added:

- setup the project Leaderboard using webpack;
- create the skeleton of the project within an HTML file;
- apply minimum styling with CSS;
- create README skeleton;

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        </div>
+      </section>
+      <section class="add-score-section sections">
+        <h2 class="title-section add-score-title">Add your score</h2>
+        <form class="form">
+          <input type="text" class="input" placeholder="Your name" />
+          <input type="number" class="input" placeholder="Your score" />
+          <button type="submit" class="btn submit-btn">Submit</button>
+        </form>
+      </section>
```
Comment (henatan99) [inline dist/index.html]: - [x] One of the objectives of this milestone is to practice JS ES6 modules, functions, syntaxes, and DOM. I think the plain HTML phrase is used to emphasize that styling using CSS is mandatory for this milestone. Kindly construct and render components in JS DOM which are children of the root HTML element. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @nicupop729 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (nicupop729) [reply dist/index.html]: Thanks a lot for the recommendation! It was taken care of!

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @nicupop729, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**